ios/Mappers.swift

```diff
@@ -506,6 +506,35 @@ class Mappers {
         default: return LogLevel.none
         }
     }
+
+    class func mapFromNetworkStatus(_ status: NetworkStatus) -> String {
+        switch status {
+        case NetworkStatus.online: return "online"
+        case NetworkStatus.offline: return "offline"
+        case NetworkStatus.unknown: return "unknown"
+        default: return "unknown"
+        }
+    }
+    
+    class func mapFromOfflineStatus(_ offlineStatus: OfflineStatus) -> NSDictionary {
+       let result: NSDictionary = [
+           "networkStatus": mapFromNetworkStatus(offlineStatus.sdk.networkStatus)
+       ]
+        return result
+    }
+    
+    class func mapFromForwaredePaymentIntent(_ paymentIntent: PaymentIntent) -> NSDictionary {
```

Comment:
I don't think we need this and can just use the existing `mapFromPaymentIntent` method